### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by KYamani**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/en/docs/setup/_index.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ The following production environment solutions table lists the providers and the
 | [Gardener](https://gardener.cloud/) | &#x2714; | &#x2714; | &#x2714; | &#x2714; | &#x2714; | [Custom Extensions](https://github.com/gardener/gardener/blob/master/docs/extensions/overview.md) |
 | [Giant Swarm](https://www.giantswarm.io/)     | &#x2714;       | &#x2714; |   &#x2714;    |     |
 | [Google](https://cloud.google.com/)           |  [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/)       | [Google Compute Engine (GCE)](https://cloud.google.com/compute/)|[GKE On-Prem](https://cloud.google.com/gke-on-prem/)       |  |  |     |  |      |       |  |
+| [Hidora](https:/hidora.com/)           |  &#x2714;       | &#x2714;| &#x2714;       |  |  |     |  |      |       |  |
```
Comment (sftim) [inline content/en/docs/setup/_index.md]: It looks like
```suggestion
| [Hidora](https://hidora.com/kubernetes-hosting/)           |  &#x2714;       | &#x2714;| &#x2714;       |  |  |     |  |      |       |  |
```
would work better.

**Comment by sftim** [comment]
Hi @KYamani 
I'd like to check something; you work for Hidora, right?

Is there someone unrelated to Hidora who can validate this change?

**Comment by KYamani** [comment]
Hi @sftim 
Thank you.
Yes I work for Hidora.


_Is there someone unrelated to Hidora who can validate this change?_
Yes sure. For example @ihorman 
How can he validate ?

**Comment by ihorman** [comment]
Hello @sftim, @KYamani 
My name is Ihor, from @Jelastic. 

> Is there someone unrelated to Hidora who can validate this change?
I confirm that change is correct.

**Comment by bradtopol** [comment]
@sftim Are you satisfied that the change has been vaiidated? If so please go ahead and approve.
Thanks!
/assign @sftim

**Pull request merged**